crates/cxx-qt-lib/src/core/qvariant/mod.rs

```diff
@@ -35,6 +35,9 @@ mod ffi {
         #[doc(hidden)]
         #[rust_name = "qvariant_clone"]
         fn construct(variant: &QVariant) -> QVariant;
+        #[doc(hidden)]
```

Comment:
Can you add a `CHANGELOG` entry ? Probably under Fixed for missing the PartialEq for QVariant?